src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,8 @@
+@mixin hand($width, $height, $bg-color) {
+  position: absolute;
+  top: 49%;
```

Comment:
why 49%? may be 50%?

Reply:
in case of 50% the hand comes too much close to bottom border and had much free space at the top)